### PR TITLE
Fix issue with 204 responses raising an error.

### PR DESCRIPTION
**Pull request by mdwelsh**
204 responses (e.g., from deleteRevision) were raising errors, this fixes that.

### CODE REVIEW

File: packages/fixie-common/src/agent.ts [hunk 1]
```diff
@@ -301,11 +301,7 @@ export class FixieAgentBase {
...
   }
 
-  public deleteRevision(revisionId: string): Promise<void> {
-    return this.client.requestJson(
-      `/api/v1/agents/${this.metadata.agentId}/revisions/${revisionId}`,
-      undefined,
-      'DELETE'
-    );
+  public deleteRevision(revisionId: string) {
+    this.client.requestJson(`/api/v1/agents/${this.metadata.agentId}/revisions/${revisionId}`, undefined, 'DELETE');
```
Comment (petersalas) [inline packages/fixie-common/src/agent.ts:305]: - Should either `return` or `await` the promise
- Should this just be calling `this.client.request`? Seems weird to invoke `requestJson` when there's no JSON

File: packages/fixie-common/src/client.ts [hunk 2]
```diff
@@ -100,6 +100,9 @@ export class FixieClientBase {
 
   async requestJson<T = Jsonifiable>(path: string, bodyData?: unknown, method?: string): Promise<T> {
     const response = await this.request(path, bodyData, method);
+    if (response.status === 204) {
```
Comment (petersalas) [inline packages/fixie-common/src/client.ts:103]: Should this be checking `Content-Type` instead?

Comment (mdwelsh) [reply packages/fixie-common/src/client.ts:103]: I don't think so. 204 has no content-type, right?

File: packages/fixie-common/src/agent.ts (hunk 1 above)
Comment (mdwelsh) [reply packages/fixie-common/src/agent.ts:305]: Yeah, I'll clean this up everywhere.

**Review by mdwelsh — COMMENTED** [review]
Thanks - PTAL

**Review by petersalas — APPROVED** [review]

**Pull request merged**